Dockerfile

```diff
@@ -35,60 +35,29 @@ RUN apt-get update && apt-get install yarn
 
 # Create app user and set WORKDIR to its home dir
 RUN adduser --ingroup staff --disabled-password --gecos "" app
-ENV APP_HOME=/home/app/pwr2
-RUN mkdir $APP_HOME
+ENV APP_HOME=/home/app
+ENV UMBRELLA_ROOT=/home/app/webapp
```

Comment:
Change it to `UMBRELLA_ROOT=/home/app/pwr2`. `webapp` is an app inside the umbrella project.